### ISSUE TITLE
Does CRNN support text line images?

### ISSUE THREAD

**Issue by thetruejacob** [FLming/CRNN.tf2#28]
I tried to run the demo of a trained model on a number of images with text separated by spaces - but only the first word was predicted. I'm a little confused by this - there are multiple words in the images, and I have checked that a space is also in the character table.

**Comment by FLming** [comment]
Of course, it's just that the pre-trained model is based on a single word. The training set can use a whole paragraph of text, but there is a theoretical upper limit for the length, and the accuracy will be greatly reduced if the length exceeds a certain length.
There is a more complex [system](https://github.com/PaddlePaddle/PaddleOCR) here, although the principles of recognition are similar.
